### ISSUE TITLE
Add support mkr plugin install

### ISSUE THREAD

**Issue by cohalz** [kazeburo/mackerel-plugin-maxcpu#3]
Failed to install plugin by mkr plugin install.

```bash
$ sudo mkr plugin install --upgrade kazeburo/mackerel-plugin-maxcpu@v0.0.5
           Downloading https://github.com/kazeburo/mackerel-plugin-maxcpu/releases/download/v0.0.5/mackerel-plugin-maxcpu_linux_amd64.zip
     error Failed to install plugin while downloading an artifact: http response not OK. code: 404, url: https://github.com/kazeburo/mackerel-plugin-maxcpu/releases/download/v0.0.5/mackerel-plugin-maxcpu_linux_amd64.zip
```

Please consider supporting `mkr plugin install`. [docs](https://mackerel.io/ja/docs/entry/advanced/make-plugin-corresponding-to-installer)

**Comment by kazeburo** [comment]
Thank you.
It's required changing package path  to support mkr plugin.
I'll do it.

**Comment by kazeburo** [comment]
I released v0.0.6.
https://github.com/kazeburo/mackerel-plugin-maxcpu/releases/tag/v0.0.6
It's support mkr plugin. Please try this.

**Comment by cohalz** [comment]
Thank you!!

```
$ sudo mkr plugin install --upgrade kazeburo/mackerel-plugin-maxcpu@v0.0.6
           Downloading https://github.com/kazeburo/mackerel-plugin-maxcpu/releases/download/v0.0.6/mackerel-plugin-maxcpu_linux_amd64.zip
           Installing /opt/mackerel-agent/plugins/bin/mackerel-plugin-maxcpu
           Successfully installed kazeburo/mackerel-plugin-maxcpu@v0.0.6
```
